envoy/tests/test_envoy.py

```diff
@@ -15,6 +15,7 @@
 CHECK_NAME = 'envoy'
 
 
+@pytest.mark.integration
```

Comment:
Should we put integration and unit tests in separate files ?

Reply:
probably, but out of scope for this PR